README.md

```diff
@@ -193,12 +193,15 @@ make build
 
 ```bash
 # Run a specific test suite or test
-./cluster-kube-apiserver-operator-tests-ext run-suite SUITE=openshift/cluster-kube-apiserver-operator/all
+./cluster-kube-apiserver-operator-tests-ext run-suite openshift/cluster-kube-apiserver-operator/all
```

Comment:
since there is no `openshift/cluster-kube-apiserver-operator/all` let's be specific and only mention `openshift/cluster-kube-apiserver-operator/serial` ?